### PR TITLE
weekly image cve patching from Sec Pipelines

### PR DESCRIPTION
**Pull request by HDYA**
weekly image cve patching from Sec Pipelines

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
       "amd64OnlyVersions": [
         "v2.0.0",
+        "v2.0.0",
```
Comment (andyliuliming) [inline vhdbuilder/packer/components.json:14]: why always have dup entries.

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -293,7 +299,8 @@
       ],
       "multiArchVersions": [
         "v1.17.0",
-        "v1.18.0"
+        "v1.18.0",
```
Comment (andyliuliming) [inline vhdbuilder/packer/components.json:302]: why keep 2 version.

File: vhdbuilder/packer/components.json [hunk 3]
```diff
@@ -124,14 +127,17 @@
       ],
       "multiArchVersions": [
         "v1.0.0.2",
-        "v1.0.0.3"
+        "v1.0.0.3",
```
Comment (andyliuliming) [inline vhdbuilder/packer/components.json:130]: is it expected to have 4 versions?

**Review by andyliuliming — CHANGES_REQUESTED** [review]

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (HDYA) [reply vhdbuilder/packer/components.json]: fixed

File: vhdbuilder/packer/components.json (hunk 3 above)
Comment (HDYA) [reply vhdbuilder/packer/components.json]: 1.0 and 1.1 are considered different minor versions, should be by design

File: vhdbuilder/packer/components.json (hunk 2 above)
Comment (HDYA) [reply vhdbuilder/packer/components.json:296]: I don't think auto patch will remove a minor version, shall we delete it manually?

Comment (HDYA) [reply vhdbuilder/packer/components.json:296]: I think it should be owner's call to remove another minor version from VHD, when they did a bump manually

**Review by andyliuliming — APPROVED** [review]

**Pull request merged**